samples/caf_sensor_manager/README.rst

```diff
@@ -32,6 +32,51 @@ This module handles the data received from the :ref:`caf_sensor_data_aggregator`
 If you are running this sample on an SoC with multiple cores, the workload simulator module (``workload_sim``) is placed on the second core.
 All communication between the cores is done using :ref:`event_manager_proxy` and Zephyr subsystem :file:`include/ipc/ipc_service.h`.
 
+Sensor stub
+===========
+
+By default, the sensor manager sample uses :ref:`sensor_sim` for sensor data simulation.
+You can select the sensor stub module instead.
+The sensor stub module has a simple implementation that relates fully to the application implementing the function to generate the sensor data.
+This sample implements a simple generator for the sensor stub that uses no floating point mathematics.
+
+For more details on the sensor stub configuration, see the :ref:`sensor_stub_config` section.
+
+Configuration
+*************
+
+|config|
+
+Single core configuration
+=========================
+
+For the MCU with multiple cores, the default configuration would use one core to simulate the sensor and the other core to process the sensor.
```

Comment:
@rakons : Is my understanding correct for these 2 sentences? I think we could have some explanation (one liner) about the single core configuration.